CHANGES/5250.feature

```diff
@@ -0,0 +1 @@
+Add requirements.yaml specification support to collection sync.
```

Comment:
Consider adding this to be a link to the galaxy docs that specify the spec. I think users would like to read "oh pulp_ansible supports this thing, <click link> oh wow here are the docs on this that I didn't even know about!"  Just a thought. The link syntax here is Sphinx "external link".

Reply:
I should have linked to the syntax, sorry https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#internal-and-external-links

Reply:
now I think I got it right
https://livesphinx.herokuapp.com/